sdk/src/tx/baseTxSender.ts

```diff
@@ -210,6 +214,22 @@ export abstract class BaseTxSender implements TxSender {
 		}
 
 		if (response === null) {
+			if (this.confirmationStrategy === ConfirmationStrategy.Combo) {
+				try {
+					const rpcResponse = await this.connection.getSignatureStatus(
+						signature
```

Comment:
should you pass in the opts sent on tx sender?

Reply:
no this is the only config option:

https://solana-labs.github.io/solana-web3.js/types/SignatureStatusConfig.html